packages/ssl/ssl.0.5.2/opam

```diff
@@ -6,15 +6,13 @@ dev-repo: "https://github.com/savonet/ocaml-ssl.git"
 bug-reports: "https://github.com/savonet/ocaml-ssl/issues"
 
 build: [
-  ["./configure" "--prefix" prefix] { os != "darwin" }
-  ["sh" "-exc"
-   "./configure --prefix %{prefix}% CPPFLAGS=\"$CPPFLAGS -I/opt/local/include -I/usr/local/opt/openssl/include\""
-  ] { os = "darwin" }
+  ["./configure" "--prefix" prefix opamconfig:cppflags]
```

Comment:
Because it is wrong. It does not *always* work and needs to be replaced by something that always works. See #6271, #6703.